control-plane/pkg/reconciler/consumergroup/controller.go

```diff
@@ -99,12 +102,32 @@ func NewController(ctx context.Context) *controller.Impl {
 	impl := cgreconciler.NewImpl(ctx, r)
 	consumerInformer := consumer.Get(ctx)
 
+	consumerGroupInformer := consumer.Get(ctx)
+
 	consumergroup.Get(ctx).Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
 	consumerInformer.Informer().AddEventHandler(controller.HandleAll(enqueueConsumerGroupFromConsumer(impl.EnqueueKey)))
 
+	globalResync := func(interface{}) {
+		impl.GlobalResync(consumerGroupInformer.Informer())
```

Comment:
can change to `consumerInformer` from line 103

Reply:
This is the consumer group reconciler isn't consumerGroupInformer what we should use?

Reply:
Yes yes. Misread the field. Changes on 105 looks good.